system-test/spanner.ts

```diff
@@ -758,6 +758,23 @@ describe('Spanner', () => {
       });
     });
 
+    describe('oids', () => {
+      it('POSTGRESQL should read pgOid values', function (done) {
```

Comment:
Should we also add a test for null pgOid value like we have added for all other types ?

Reply:
I added a test for reading a null value. As for writing, OIDs are never meant to be written, which is why there are no insert tests.